frontend/src/lib/gql/gql-client.ts

```diff
@@ -215,7 +203,12 @@ class GqlClient {
     // An error's stacktrace points to where it was instantiated (i.e. in urql),
     // but it's far more interesting (particularly when debugging how errors affect our app) to know when and where errors are getting thrown, namely HERE.
     // So, we new up our own error to get the more useful stacktrace.
-    throw new AggregateError(error.graphQLErrors, error.message ?? error.cause);
+    const aggError = new AggregateError(error.graphQLErrors, error.message ?? error.cause);
+    if (!delayThrow) {
+      throw aggError;
+    }
+    // We can't throw errors here, because errors thrown in wonka/an exchange kill the frontend.
+    setTimeout(() => { throw aggError; });
```

Comment:
@hahn-kev Sadly, this isn't good enough. We have to throw everything that isn't a `Redirect` in `setTimeout`, because SvelteKit will only handle `Redirect`'s. And `Redirect`'s are not the only kind of `error.networkError`'s. So, for example, the GQL 500 sandbox button now breaks the app again.